### PR TITLE
Update files with PEP8 standards in folder dojo/tools #001

### PR DESCRIPTION
**Pull request by ajtortolero**
Update files with PEP8 standars in folders dojo/tools/

Folder affected
dojo/tools/dawnscanner
dojo/tools/dependency_check
dojo/tools/dependency_track
dojo/tools/detect_secrets
dojo/tools/dockerbench

### CODE REVIEW

File: dojo/tools/dependency_track/parser.py [hunk 1]
```diff
@@ -242,16 +345,15 @@ def get_description_for_scan_types(self, scan_type):
...
         if file is None:
             return list()
 
         # Load the contents of the JSON file into a dictionary
         data = file.read()
         try:
-            findings_export_dict = json.loads(str(data, 'utf-8'))
-        except:
+            findings_export_dict = json.loads(str(data, "utf-8"))
+        except BaseException:
```
Comment (Maffooch) [inline dojo/tools/dependency_track/parser.py:356]: Is this an accurate exception?

Comment (ajtortolero) [reply dojo/tools/dependency_track/parser.py:356]: Hi @Maffooch, flake8 applies this change to comply with this [rule](https://www.flake8rules.com/rules/E722.html) and according to the [BaseException](https://docs.python .org/ 3/library/exceptions.html#BaseException) documentation  is a base class for any exception.

Comment (Maffooch) [reply dojo/tools/dependency_track/parser.py:356]: Please change the use of `BaseException` to the standard catch all `Exception`

Reason for this is that the `BaseException` class will also catch system exits and keyboard interrupts that may be not be desired/expected. The python docs you linked also mention that the `BaseException` class should not be directly extended (and therefore not directly used either) but that the `Exception` be used instead.

> The base class for all built-in exceptions. It is not meant to be directly inherited by user-defined classes (for that, use [Exception](https://docs.python.org/3/library/exceptions.html#Exception)).

Comment (Maffooch) [inline dojo/tools/dependency_track/parser.py:356]:
```suggestion
        except Exception:
```

**Review by Maffooch — CHANGES_REQUESTED** [review]

File: dojo/tools/dependency_track/parser.py (hunk 1 above)
Comment (ajtortolero) [reply dojo/tools/dependency_track/parser.py]: Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8282/commits/2de95f283b3df8d72cc1577b35f2e202386dd84c

**Review by mtesauro — APPROVED** [review]
Approved

**Review by blakeaowens — APPROVED** [review]

**Review by alles-klar — APPROVED** [review]

**Pull request merged**